nixos/modules/hardware/video/nvidia.nix

```diff
@@ -287,6 +290,10 @@ in
               assertion = !(nvidiaEnabled && cfg.datacenter.enable);
               message = "You cannot configure both X11 and Data Center drivers at the same time.";
             }
+            {
+              assertion = cfg.open != null;
+              message = "`hardware.nvidia.open` must be set when using driver versions >= 560.";
```

Comment:
"`hardware.nvidia.open` must be set" might be misinterpreted by some as "`hardware.nvidia.open` must be **`true`**".